gulpfile.js

```diff
@@ -1,16 +1,30 @@
 var gulp = require( 'gulp' );
 var requireDir = require( 'require-dir' );
+var runSequence = require('run-sequence');
 
 requireDir( './gulp' );
 
-gulp.task( 'watch-compile', [ 'compile' ], function () {
+function watchAndRebuild() {
+	var watcher = gulp.watch( ['./compile/**/*'], [ 'build' ] );
+	watcher.on('change', function(event) {
```

Comment:
Removed excessive logging. I just haven't deleted it since debugging, when I found out that it rebuilds too often during the initialisation because of asynchrony. Agree that there is no need of that